mono/eglib/goutput.c

```diff
@@ -40,6 +44,61 @@ static GPrintFunc stdout_handler, stderr_handler;
 static void default_stdout_handler (const gchar *string);
 static void default_stderr_handler (const gchar *string);
 
+static GAbortFunc internal_abort_func;
+
+void
+g_assertion_disable_global (GAbortFunc abort_func)
+{
+	internal_abort_func = abort_func;
+}
+
+void
+g_assert_abort (void)
+{
+	if (internal_abort_func)
+		internal_abort_func ();
+	else
+		abort ();
+}
+
+static int
+async_safe_vfprintf (int handle, gchar const *format, va_list args)
+{
+	char __buff [256];
```

Comment:
Keep the size of `1024` from https://github.com/mono/mono/pull/12125/files#diff-0dec0d6f1c59568eb9e18b6906c315edL10.

Reply:
Ok, let's keep `256` then.

Reply:
Last night's overnight stress didn't get any crashes, so I'm pretty comfortable calling 1024 safe.